webapp/src/components/admin_settings/custom_attribute/custom_attribute.jsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import UsersInput from '../users_input';
+
+const {formatText, messageHtmlToComponent} = window.PostUtils;
+
+export default class CustomAttribute extends React.Component {
+    static propTypes = {
+        id: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+        users: PropTypes.array,
+        groups: PropTypes.array,
+        hideDelete: PropTypes.bool,
+        markdownPreview: PropTypes.bool,
+        onDelete: PropTypes.func,
+        onChange: PropTypes.func.isRequired,
+        actions: PropTypes.shape({
+            getProfilesByIds: PropTypes.func.isRequired,
+        }).isRequired,
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            name: this.props.name,
+            groups: this.props.groups,
+            error: null,
+        };
+
+        this.initUsers();
+    }
+
+    // initUsers fetches user profiles for the users ids passed in props
+    async initUsers() {
+        if (!this.props.users || !this.props.users.length) {
+            return;
+        }
+
+        const profiles = await this.props.actions.getProfilesByIds(this.props.users);
+
+        let users = profiles.data;
+
+        if (users.length !== this.props.users.length) {
+            //Check if all ids where returned.
+            //MM-Redux removes the current admin user from the result
```

Comment:
```suggestion
            // mattermost-redux removes the current admin user from the result
```

Might we also link to the offending code in question to help the future issue?

Reply:
Like a github link ? 

https://github.com/mattermost/mattermost-redux/blob/5f5a8a5007661f6d54533c2b51299748338b5a65/src/actions/users.ts#L340